test_scripts/Polices/Validation_of_PolicyTables/075_ATF_usage_and_error_counts_update_count_of_iap_buffer_full.lua

```diff
@@ -14,25 +14,27 @@
 -- Expected result:
 -- SDL must: increment "count_of_iap_buffer_full" section value of Local Policy Table.
 ---------------------------------------------------------------------------------------------
-
---[[ General configuration parameters ]]
-Test = require('connecttest')
-local config = require('config')
 config.defaultProtocolVersion = 2
 config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
 
 --[[ Required Shared libraries ]]
 local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
 local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+--[[ General configuration parameters ]]
+Test = require('connecttest')
+local config = require('config')
 require('user_modules/AppTypes')
 
 --[[ Local Variables ]]
-
+local hmi_iap_buffer_full = 5
 local TESTED_DATA = {
   policy_table = {
     usage_and_error_counts = {
       count_sync_out_of_memory = 0,
-      count_of_iap_buffer_full = 5,
+      count_of_iap_buffer_full = hmi_iap_buffer_full,
```

Comment:
@dboltovskyi, thank you for notice, please ignore previous update and check commit 370fbac7ae94cb3065fcdcf5b6dd4722dbbcb632